### ISSUE TITLE
BT segfault

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#2509]
```
(gdb) bt
#0  0x00007f049e2b1100 in FreeNode (node=0xcde7c3e44a57619e) at /bigfiles/contrib/unvanquished/Unvanquished/src/sgame/sg_bot_parse.cpp:1603
#1  0x00007f049e2b0d54 in FreeBehaviorTree (tree=0x7f0499063bd0) at /bigfiles/contrib/unvanquished/Unvanquished/src/sgame/sg_bot_parse.cpp:1626
#2  0x00007f049e2b0e51 in FreeTreeList (list=0x7f049e5b2560 <treeList>) at /bigfiles/contrib/unvanquished/Unvanquished/src/sgame/sg_bot_parse.cpp:1478
#3  0x00007f049e2a1a47 in G_BotCleanup () at /bigfiles/contrib/unvanquished/Unvanquished/src/sgame/sg_bot.cpp:610
#4  0x00007f049e3055ab in G_ShutdownGame () at /bigfiles/contrib/unvanquished/Unvanquished/src/sgame/sg_main.cpp:721
```

This is fully reproducible, and is caused by a problem in the BT files in this case. Game should still not crash for this.
This crash happens when closing the game, there's another one when the game starts. More details later.

**Comment by necessarily-equal** [comment]
You probably should include the exact BT used if you still can. I don't think I've seen this bug before

**Comment by bmorel** [comment]
Me neither. Same for the segfaults I had yesterday when *opening* a BT with an error.

[bt.zip](https://github.com/Unvanquished/Unvanquished/files/10989709/bt.zip)

The BT I had with the crash are in the zip file of this message.

**Comment by bmorel** [comment]
```
Warn: Invalid token on line 38 found: changeGoal 
Warn: Failed to parse child node of condition on line 36 
Warn: Could not load behavior default_aliens on line 7 
Warn: Failed to parse child node of condition on line 5 
Warn: Problem when loading behavior tree default, trying default 
Warn: Crashed with signal 11: Segmentation fault 
Waiting for the VM thread... 
----- Server Shutdown ----- 
```

**Comment by slipher** [comment]
Fixed in #2575.
